apcd-cms/src/apps/utils/registrations_data_formatting.py

```diff
@@ -62,7 +62,7 @@ def format_phone_number(num):
         'notif': reg_cont[2],
         'role': reg_cont[3],
         'name': reg_cont[4],
-        'phone': format_phone_number(reg_cont[5]),
+        'phone': format_phone_number(reg_cont[5]) if reg_cont[5] else None,
```

Comment:
So to confirm, `None` is an acceptable value for the `phone` field?

Reply:
I'll check with UTH to confirm.

Reply:
Confirmed with them, None is good!